packages/medusa/src/loaders/index.ts

```diff
@@ -22,7 +22,6 @@ import { SubscriberLoader } from "./helpers/subscribers"
 type Options = {
   directory: string
   expressApp: Express
-  isTest: boolean
```

Comment:
Note: removed this as it's a required option for the `loaders` function but didn't find it used anywhere